deployment/kubernetes/flood-api.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flood-api-deployment
+  namespace: apps-flood-api
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: flood-api
+  template:
+    metadata:
+      namespace: apps-flood-api
+      labels:
+        app: flood-api
+    spec:
+      containers:
+      - image: ghcr.io/openearthplatforminitiative/flood-api:0.3.0
+        name: flood-api
+        ports:
+        - containerPort: 8080
+        env:
+        - name: DB_NAME
+          value: flood_data
+        - name: DB_ENDPOINT
+          valueFrom:
+            secretKeyRef:
+              name: database-connection-secrets
```

Comment:
I assume this secret is local in the namespace of the application, so we won't get conflicts if other applications need this?

Reply:
Yes, see line 84 of this file:
```
writeConnectionSecretToRef:
    namespace: apps-flood-api
    name: database-connection-secrets
```

It's not allowed to specify a namespace in `secretKeyRef`, I assume this is because you're only allowed to access secrets in your own namespace anyway